modules/ROOT/pages/sap/sap-connector.adoc

```diff
@@ -36,74 +35,75 @@ and resources on external systems, such as Salesforce, SAP, ServiceNow, and Twit
-* Power user - Read xref:sap/sap-connector-xml-maven.adoc[XML and Maven Support], 
-the xref:sap/sap-connector-config-topics.adoc[Additional Configuration Information] topic, 
-and xref:sap/sap-connector-examples.adoc[Examples].
+* Power user - Read the xref:sap/sap-connector-xml-maven.adoc[XML and Maven Support], 
+xref:sap/sap-connector-config-topics.adoc[Additional Configuration Information], 
+and xref:sap/sap-connector-examples.adoc[Examples] topics.
 
 == SAP Terminology
 
-ABAP (advanced business application programming)::
+ABAP (advanced business application programming) +
 SAP proprietary programming language and part of the NetWeaver platform for building business applications.
 
-ALE (application link enabling)::
+ALE (application link enabling) +
 Sets up distributed applications and integrated installation of SAP systems. This involves business-driven message exchange using consistent data across loosely linked SAP apps. Apps are integrated through synchronous and asynchronous communication, rather than by use of a central database.
 
-BAPI (business application programming interface)::
+BAPI (business application programming interface) +
 The function module that fulfills design criteria, such as:
 ** Implement a method of a SAP business object.
 ** Maintain a static interface through different versions of the SAP system.
 ** Enable remotely.
 ** Run to completion with or without user interaction.
 ** Handle errors.
 
-FM (function module)::
+FM (function module) +
 Procedures defined in the ABAP language of SAP that allow the encapsulation and reuse of global functions in SAP.
 
-IDoc (Intermediate Document)::
+IDoc (Intermediate Document) +
 Standard SAP format for electronic data interchange between SAP systems. Different message types (such as delivery notes or purchase orders) generally correspond to different special formats, known as IDoc types. Multiple message types with related content can, however, be assigned to a single IDoc type.
 
-Java API::
+Java API +
 Handles dynamic metadata lookup and caching. The Java API implements the `JCO.Function` container used for parameters and/or tables for the SAP BAPI function module in Java. Java apps are built on top of the Java API.
 
-JNI (Java native interface)::
-Originally, SAP created libraries in C language to allow direct RFC calls to SAP, to manipulate with data. JCo wraps C libraries in Java to provide platform-native access into the SAP system. RFC middleware uses RFC library through the JNI Layer.
+JNI (Java native interface) +
+Originally, SAP created libraries in C language to allow direct RFC calls to SAP, to manipulate with data. JCo wraps C libraries in Java to provide platform-native access into the SAP system. RFC middleware uses the RFC library through the JNI Layer.
```

Comment:
I don't understand how ", to manipulate with data" works with the rest of the sentence? To manipulate what?